hack/commit-msg-verify.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+# PULL_BASE_SHA and PULL_PULL_SHA are set in prow job
+PULL_BASE_SHA="${PULL_BASE_SHA:-$(git merge-base master HEAD)}"
+PULL_PULL_SHA="${PULL_PULL_SHA:-HEAD}"
+
+pattern="^[A-Z]+-[0-9]+ \| (feat|fix|docs|style|refactor|test|chore|build|ci|perf) : .*$"
```

Comment:
Will we have a document/announcement about this format?

Reply:
we already do, are you telling me do didnt read our Contribution guide? TUT TUT TUT

https://github.com/openshift/rosa/blob/master/CONTRIBUTING.md

Also this is why we need this job, no one reads the documentation :rofl:

Reply:
:laughing:  i am ashamed